From b97eadcf6ba612b717478bc38becc184e6205c2d Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:04:22 -0400
Subject: [powerpc] eeh: Add missing procfs entry for PowerNV

Message-id: <1410545655-205645-34-git-send-email-dzickus@redhat.com>
Patchwork-id: 94071
O-Subject: [RHEL7 PATCH 033/626] powerpc/eeh: Add missing procfs entry for PowerNV
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 144136dd7a25a0ca4d86685f872168502f91f337
Author: Mike Qiu <qiudayu@linux.vnet.ibm.com>
Date:   Tue Aug 6 22:25:14 2013 -0400

    powerpc/eeh: Add missing procfs entry for PowerNV

    The procfs entry for global statistics has been missed on PowerNV
    platform and the patch is going to add that.

    Signed-off-by: Mike Qiu <qiudayu@linux.vnet.ibm.com>
    Acked-by: Gavin Shan <shangw@linux.vnet.ibm.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kernel/eeh.c b/arch/powerpc/kernel/eeh.c
index 8d4bf9e..40829aa 100644
--- a/arch/powerpc/kernel/eeh.c
+++ b/arch/powerpc/kernel/eeh.c
@@ -1061,7 +1061,7 @@ static const struct file_operations proc_eeh_operations = {
 
 static int __init eeh_init_proc(void)
 {
- if (machine_is(pseries))
+ if (machine_is(pseries) || machine_is(powernv))
   proc_create("powerpc/eeh", 0, NULL, &proc_eeh_operations);
  return 0;
 }
-- 
1.7.1